tests/testthat/test-workflow-bbi.R

```diff
@@ -305,5 +305,160 @@ withr::with_options(list(
     run_times <- model_summaries(mods) %>% check_run_times(.wait = FALSE) %>% suppressWarnings()
     expect_equal(dim(run_times), c(3, 3))
   })
+
+  describe("run_nmtran", {
+    it("nmtran_setup: executable and nonmem version", {
+      mod1 <- read_model(file.path(MODEL_DIR_BBI, "1"))
+      # Using model object, looks for bbi.yaml
+      nmtran_specs <- nmtran_setup(mod1)
+      # Confirm executable
+      expect_equal(nmtran_specs$nmtran_exe, "/opt/NONMEM/nm74gf/tr/NMTRAN.exe")
+      # Confirm NONMEM version
+      expect_equal(nmtran_specs$nonmem_version, "nm74gf")
```

Comment:
These will fail if either `BBR_TESTS_NONMEM_DIR` or `BBR_TESTS_NONMEM_VERSION` is set to something other than the default value. Same for the assertions on lines 329 and 331. The existence of those environment variables is motivated by Metworx updates: [773873e6](https://github.com/metrumresearchgroup/bbr/commit/773873e69d23ea2d5371ff8f60f20257323c5713) (test-workflow-bbi.R: let runner override bbi\_init values, 2024-01-03, gh-630).

I don't see a way around having these new tests tightly coupled to Metworx details.

My suggestion:

-   put these `run_nmtran` tests in a separate file (see comment above)
-   skip if not on Metworx, as before
-   also skip if the `/opt/NONMEM/{version}` directory for any NONMEM installations used by the test do not exist